bool absolute
geometry_msgs/PoseStamped query
---
bool result